lib/ViewModels/TerriaViewer.js

```diff
@@ -303,6 +309,29 @@ function changeFogSettings(viewer) {
   }
 }
 
+function changeQuality(viewer) {
+  if (defined(viewer.terria.cesium)) {
+    const terria = viewer.terria;
+
+    // 0 - defaults
+    // 1 - devicePixelRatio as resolutionScale
+    // 2 - increase maximumScreenSpaceError
+    switch (viewer.terria.quality) {
+      case 1:
+        terria.cesium.viewer.resolutionScale = devicePixelRatio;
```

Comment:
After changing `resolutionScale`, you need to call `terria.cesium.viewer.resize()` for it to take effect.

Reply:
Done for any change now